app/flows/check_travel_during_coronavirus_flow/outcomes/_fully_vaxed.erb

```diff
@@ -12,17 +12,35 @@
 
   <p class="govuk-body">You do not need to take any COVID-19 tests before you travel to England or after you arrive.</p>
 
-  <p class="govuk-body">You must be able to prove that you’ve been fully vaccinated. If you live in England, you can prove your vaccination status using the <a href="https://www.gov.uk/guidance/nhs-covid-pass" class="govuk-link">NHS COVID Pass</a>.</p>
+  <p class="govuk-body">You must <a href="https://www.gov.uk/provide-journey-contact-details-before-travel-uk" class="govuk-link">complete a passenger locator form</a> in the 48 hours before you arrive in England.</p>
```

Comment:
Not a blocker: This should be a relative link. i.e. `<a href="/provide-journey-contact-details-before-travel-uk" class="govuk-link">`

Reply:
oops, copy paste error, thanks!